src/NuGet.Services.Metadata.Catalog.Monitoring/NuGet.Services.Metadata.Catalog.Monitoring.csproj

```diff
@@ -0,0 +1,122 @@
+      <HintPath>..\packages\NuGet.Protocol.Core.v3.4.0.0\lib\net45\NuGet.Protocol.Core.v3.dll</HintPath>
+    </Reference>
+    <Reference Include="NuGet.Versioning, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\packages\NuGet.Versioning.4.0.0\lib\net45\NuGet.Versioning.dll</HintPath>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.IdentityModel" />
+    <Reference Include="System.IO.Compression" />
+    <Reference Include="System.Net.Http.WebRequest" />
+    <Reference Include="System.Security" />
+    <Reference Include="System.ServiceModel" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="LoggingUtility.cs" />
+    <Compile Include="Model\NonhijackableV2HttpMessageHandlerWrapper.cs" />
+    <Compile Include="Model\PackageTimestampMetadata.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Providers\NonhijackableV2HttpHandlerResourceProvider.cs" />
+    <Compile Include="Providers\PackageTimestampMetadataResourceV2Provider.cs" />
+    <Compile Include="Providers\PackageTimestampMetadataResourceV3Provider.cs" />
+    <Compile Include="Resources\IPackageTimestampMetadataResource.cs" />
+    <Compile Include="Resources\PackageTimestampMetadataResourceV2.cs" />
+    <Compile Include="Resources\PackageTimestampMetadataResourceV3.cs" />
+    <Compile Include="ValidationCollector.cs" />
+    <Compile Include="Validation\Result\EndpointValidationResult.cs" />
+    <Compile Include="Validation\Result\PackageValidationResult.cs" />
+    <Compile Include="Validation\Result\ValidationResult.cs" />
+    <Compile Include="Validation\Test\Endpoint.cs" />
+    <Compile Include="Validation\Test\EndpointFactory.cs" />
+    <Compile Include="Validation\Test\EndpointValidator.cs" />
+    <Compile Include="Validation\Test\PackageValidator.cs" />
+    <Compile Include="Validation\Test\RegistrationValidators.cs" />
+    <Compile Include="Validation\Test\ValidationContext.cs" />
+    <Compile Include="Validation\Test\ValidationTestUtility.cs" />
+    <Compile Include="Validation\Test\Validator.cs" />
+    <Compile Include="Validation\Test\ValidatorFactory.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Catalog\NuGet.Services.Metadata.Catalog.csproj">
+      <Project>{e97f23b8-ecb0-4afa-b00c-015c39395fef}</Project>
+      <Name>NuGet.Services.Metadata.Catalog</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="app.config" />
+    <None Include="packages.config" />
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
```

Comment:
Please compare to existing projects (e.g. NuGet.Indexing.csproj). In short, you need to conditionally import `sign.targets`.